presto-main/src/main/java/com/facebook/presto/metadata/BuiltInTypeAndFunctionNamespaceManager.java

```diff
@@ -1178,6 +1181,19 @@ else if (function instanceof SqlInvokedFunction) {
                     sqlFunction.getVersion(),
                     sqlFunction.getComplexTypeFunctionDescriptor());
         }
+        else if (function instanceof ParametricScalar) {
```

Comment:
Why is this new block required?

Reply:
We are storing functions stats header inside FunctionMetadata.